data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -266,7 +266,10 @@ def _create_downloadable_data(
                     number_of_decimals=constants.TILES_ROUND_NUM_DECIMALS,
                 )
 
-        return df
+        # sort by tract id
+        df_sorted = df.sort_values(self.GEOID_TRACT_FIELD_NAME)
```

Comment:
@emma-nechamkin pointed out to me the other day that this is not a stable sort (if there's a tie in any values, it doesn't guarantee returning the same order every time).

However, I don't think that matters here because there should only be one row per tract, so no ties.